resource/css/styles.css

```diff
@@ -777,10 +773,14 @@ span.xl-pref-label > img {
 
 /* feedback page
  ***************************/
-#feedback-fields > .missing-value {
+#feedback-fields > p .missing-value {
   border: 1px solid red;
```

Comment:
I think it would be better to use the color #D95F8A that Skosmos uses for deprecated concepts, see https://github.com/NatLibFi/Skosmos/blob/a201ace07d0aad5c3ddf4c2dbbbe045f4834ff4c/resource/css/styles.css#L1209-L1212
(I didn't check how that would look though)

Reply:
```
#feedback-fields > p .missing-value {
  border: 1px solid red;
}
```

Changed to ->

```
#feedback-fields > p .missing-value {
  border: 2px solid #D95F8A;
}
```

Contrast tests for text box with new border style passed
(https://contrastchecker.com/)

Reply:
Asterisks added